status.go

```diff
@@ -118,3 +118,16 @@ func (r *StatusResp) WriteTo(w *Writer) error {
 
 	return w.writeCrlf()
 }
+
+// ErrStatusResp can be returned by a Handler to replace the default status
```

Comment:
Handler is a server thing. Maybe we can document that `imap.ErrStatusResp` can be used in `server.Handler` instead?

Reply:
server.Handler.Handle code already references ErrStatusResp. I updated it to reference imap.ErrStatusResp and also fixed ErrStatusResp doc to reference server.Handler. Is that better?

Reply:
Looks good!